### ISSUE TITLE
Smruthi: it looks like chatgpt made it

### ISSUE THREAD

**Issue by mrjkey** [Tris20/FairFareFinder#21]
Make the UI prettier

**Comment by Tris20** [comment]
Agree. do magic please. that table looks like poop. make it look like chocolate

**Comment by bneenb** [comment]
I'm not a designer. Its already much prettier than I expected!

**Comment by bneenb** [comment]
Made a prototype for how to display locations.
Still basic colors, almost no animation. But can be extended.
![image](https://github.com/Tris20/FairFareFinder/assets/9251011/48338d34-5ff1-4f00-a38d-4bc1e993ee3c)

Future plan would be that there are little symbols for fast links to flights/accommodation/etc...
If you click on the card/box it will extend and will show a summary of information of these things.

This is a sketch:
![image](https://github.com/Tris20/FairFareFinder/assets/9251011/46f2b07f-b6f4-4585-930a-efc3378d4d51)

Let me know what you think!

**Comment by Tris20** [comment]
![image](https://github.com/Tris20/FairFareFinder/assets/6933318/b5078909-6fff-4f6e-863c-fe13e340f108)

Taking inspiration from bneenb
